tests/Browser/PilotPassportChallengeTest.php

```diff
@@ -0,0 +1,145 @@
+        $this->browse(function (Browser $browser) {
+            $browser->visit('/pilot_passport')
+                    ->assertSee('What is the ZTL Pilot Passport Challenge?');
+        });
+    }
+
+    public function test_home_authenticated(): void {
+        $this->loginSamplePilot();
+        $this->browse(function (Browser $browser) {
+            $browser->visit('/pilot_passport')
+                    ->assertSee('Welcome, ');
+        });
+    }
+    
+    public function test_enrollments_view(): void {
+        $this->browse(function (Browser $browser) {
+            $browser->visit('/pilot_passport')
+                    ->clickLink('Enrollments')
+                    ->assertSee('Select a path below and enroll to get started.');
+        });
+    }
+
+    public function test_enroll(): void {
+        $this->browse(function (Browser $browser) {
+            $browser->visit('/pilot_passport')
+            ->clickLink('Enrollments')
+            ->press('@enroll_1')
+            ->assertSee('You are now enrolled in the ZTL Pilot Passport program!');
+        });
+    }
+
+    public function test_passport_book_view(): void {
+        $this->browse(function (Browser $browser) {
+            $browser->visit('/pilot_passport')
+                    ->clickLink('Passport Book')
+                    ->assertSee('Visited on: ');
+        });
+    }
+
+    public function test_achievements_view(): void {
+        $this->browse(function (Browser $browser) {
+            $browser->visit('/pilot_passport')
+                    ->clickLink('Achievements')
+                    ->assertSee('Completed on: ');
+        });
+    }
+
+    public function test_settings_view(): void {
+        $this->browse(function (Browser $browser) {
+            $browser->visit('/pilot_passport')
+                    ->clickLink('Settings')
+                    ->assertSee('ZTL understands that some pilots may want to participate');
+        });
+    }
+
+    public function test_change_privacy_settings(): void {
+        $this->browse(function (Browser $browser) {
+            $browser->visit('/pilot_passport')
+                    ->clickLink('Settings')
+                    ->radio('@privacy', '2')
+                    ->press('Save Settings')
+                    ->assertSee('Your privacy preferences have been saved.');
+        });
+    }
+
+    public function test_disenroll_purge(): void {
+        $this->browse(function (Browser $browser) {
+            $browser->visit('/pilot_passport')
+                    ->clickLink('Settings')
+                    ->press('@purge_data')
+                    ->whenAvailable('.modal', function (Browser $modal) {
+                        $modal->type('@confirm', 'confirm - purge all')
+                            ->press('Continue');
+                    })
+                    ->waitFor('.navbar', 30)
+                    ->assertSee('You have been successfully logged out');
+        });
+    }
```

Comment:
Same here, can we test to make sure the disenroll and purge actually worked? If this logs the user out without actually deleting anything, this would still be successful which would be misleading

Reply:
Sure, easy enough. I added a second test just to check to see if the data has been purged.